specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2894,17 +2890,32 @@
           "type": "boolean",
           "description": "Specifies whether this compute instance is running on the latest operating system image."
         },
-        "rebootPending": {
-          "type": "boolean",
-          "description": "Specifies whether this compute instance is pending for reboot to finish os patching."
-        },
+        "osPatchingStatus": {
+          "readOnly": true,
+          "description": "Metadata about the os patching.",
+          "$ref": "#/definitions/osPatchingStatus"
+        }
+      }
+    },
+    "osPatchingStatus": {
+      "type": "object",
+      "description": "Returns metadata about the os patching.",
+      "properties": {
         "patchStatus": {
           "type": "string",
           "description": "The os patching status."
         },
-        "latestUpdateTime": {
+        "latestPatchTime": {
           "type": "string",
           "description": "Time of the lastest os patching."
+        },
+        "rebootPending": {
+          "type": "boolean",
+          "description": "Specifies whether this compute instance is pending for reboot to finish os patching."
+        },
+        "rebootRequiredTime": {
```

Comment:
What is this property needed for? I thought this was rather a handshake between AMl and flex than customer-facing.

Reply:
renamed as discussed offline